### DISCUSSION TITLE
Simulation mode - error in starting the simulation

### DISCUSSION THREAD

**Discussion by a-m-a-n-d-a** [asreview/asreview#872] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Hi, 

I seem be facing an error in starting the simulation. I have a labelled dataset entitled 'enrico_test'. When I attempt to run the simulation, I get the following error.

C:\Users\Amanda\Desktop\test>asreview simulate enrico_test --state_file simulation_results.h5 --seed 100

FileNotFoundError: File, URL, or dataset does not exist: 'enrico_test'

When I specified the file type, it gives me anoher error.

asreview simulate enrico_test.xlsx ––state_file simulation_results.h5 ––seed 100 
OR
asreview simulate enrico_test.csv ––state_file simulation_results.h5 ––seed 100 

TypeError: object of type 'NoneType' has no len()

I am not sure what went wrong here. I have upgraded asreview to the newest version. Any help on this matter is very much appreciated.

Amanda

**Comment by SagevdBrand** [comment]
Hi @a-m-a-n-d-a !

Just to make sure, the data file in the test folder should indeed have exactly the name + the extension. It seems that you solved that issue correctly in your second and third try. 

However, it looks like you only used one `-` in the state_file command and the seed command?
Could you try the following command perhaps (after navigating to you test folder).

`asreview simulate enrico_test.csv --state_file simulation_results.h5 --seed 100`

Please let me know if you need more help!

**Reply by a-m-a-n-d-a** [reply]
Hi Sofie,

Thank you for helping out.

I tried the command (with two --) and the same error persisted:

TypeError: object of type 'NoneType' has no len()

**Reply by SagevdBrand** [reply]
Would it be possible to send me your test file perhaps? Then I can try out a few things :)
You can also send it to asreview@uu.nl!

**Reply by a-m-a-n-d-a** [reply]
Sure. I have emailed the test file to asreview@uu.nl . Thank you for your help!

**Comment by SagevdBrand** [comment, marked as answer]
It appeared that the column with the inclusion labels was called `debug_label` which should have been `label_included`. 
Simulation mode looks for a column with the name `label_included`, but when it is not present then indeed the error:
`TypeError: object of type 'NoneType' has no len()` pops up!

**Reply by a-m-a-n-d-a** [reply]
I see.  Appreciate the help in troubleshooting :D
